index.js

```diff
@@ -31,9 +31,7 @@ function runWebpack({env, config, webpackConfig}) {
 			if (!config.isSSR) {
 				console.log(formatStats(stats, config.destPath));
 			}
-
-			// give time to webpack bundle analyzer to output stats
-			setImmediate(resolve);
+			resolve();
```

Comment:
I think this might create problems, `setImmediate(resolve);` is fine or that creates any issues?

Reply:
This was no longer needed as it was fixed in bundle analyzer itself: https://github.com/webpack-contrib/webpack-bundle-analyzer/issues/232